soc/riscv/riscv-ite/it8xxx2/Kconfig.soc

```diff
@@ -105,4 +105,13 @@ config SOC_IT8XXX2_EXCEPTIONS_IN_RAM
 	  Flash. This can significantly improve performance when under I-cache
 	  pressure.
 
+config SOC_IT8XXX2_SHA256_HW_ACCELERATE
+	bool "HW SHA256 calculation"
+	default n
+	help
+	  IT8XXX2 HW support sha256 calculation, and its calculation is faster than FW.
+	  We place SHA256 message, hash and key data (total 512bytes) in RAM.
+	  If we enable this config, because HW limits, the sha256 data must place in
+	  top 4KB of RAM.
```

Comment:
Does "top" here refer to the first block at 0x80100000? I would usually refer to that as the bottom- it's probably clearer to call it the "first" or "last" instead of "top" or "bottom."

Reply:
Yes, it means first block at 0x80100000 (0x80100000 ~ 0x80101000).
Ok, I updated.